corehq/apps/users/signals.py

```diff
@@ -51,3 +57,4 @@ def connect_user_signals():
     couch_user_post_save.connect(sync_user_phone_numbers, dispatch_uid="sync_user_phone_numbers")
     commcare_user_post_save.connect(apply_correct_demo_mode,
                                     dispatch_uid='apply_correct_demo_mode')
+    commcare_user_post_save.connect(remove_test_cases, dispatch_uid='remove_test_cases')
```

Comment:
So if the original mobile worker is deleted, then the copies of their cases are also deleted? Maybe add a comment here explaining that (and if I've misunderstood, please definitely add a comment).

Reply:
@kaapstorm 
With "original mobile worker" I presume you mean "the owner of the original case"? A copied case is not deleted when the "original owner" is deleted, only when the owner of the copied case is deleted. The copied case is completely detached from the original case. 

This `remove_test_cases` only tags the copied cases belonging to this particular mobile worker as "deleted" when the user is deactivated. [This](https://github.com/dimagi/commcare-hq/blob/3e62cb7edab54575023b316b868e1ae5e8177d7e/corehq/apps/users/tasks.py#L398) is the celery task that executes when the mobile worker is deactivated.

Reply:
When a mobile worker is deactivated, we already delete their cases, don't we? Are we doing something different here?